.gitignore

```diff
@@ -3,29 +3,8 @@
 tags
 *.swp
 
-################################################################################
-# https://raw.githubusercontent.com/github/gitignore/master/Autotools.gitignore
-################################################################################
-
-# http://www.gnu.org/software/automake
-
-Makefile.in
-/ar-lib
-/test-driver
-
-# http://www.gnu.org/software/autoconf
-
-/autom4te.cache
-/autoscan.log
-/autoscan-*.log
-/aclocal.m4
-/compile
-/config.h.in
-/config.guess
-/config.sub
-/configure
-/configure.scan
-/depcomp
-/install-sh
-/missing
-/stamp-h1
+# We recommend building in a subdirectory called build.
+# If you chose a different directory name,
+# it is up to you to arrange for it to be ignored by git,
+# e.g. by listing your directory in .git/info/exclude.
+/build
```

Comment:
Since Meson 0.57.0, it will automatically create e.g. `builddir/.gitignore` with the contents `*`, so that you don't need to deal with this.

Reply:
Our minimum meson version is 0.45, which is in Ubuntu 18.04 that’s still supported until 2023. We can probably simplify this eventually, but not just now.